### ISSUE TITLE
They blocked javascript for .html files on chromebook.

### ISSUE THREAD

**Issue by enziviox** [Blobby-Boi/ExtHang3r#43]
Can't use html files.
Sorry to open another issue. Try to fix the data:text or make a github.io version please and thank you.

**Comment by bcrhbrhcdb** [comment]
i got you

**Comment by bcrhbrhcdb** [comment]
https://bcrhbrhcdb.github.io/ExtHang3r/

**Comment by enziviox** [comment]
Thanks but GitHub just got blocked. Might get unblocked though they usually do this.

> https://bcrhbrhcdb.github.io/ExtHang3r/

**Comment by bcrhbrhcdb** [comment]
is vercle?

**Comment by enziviox** [comment]
Yep. Vercel is blocked as well.

**Comment by bcrhbrhcdb** [comment]
try this: file:///media/archive/ExtHang3r-main%20(1).zip/ExtHang3r-main/index.html

**Comment by enziviox** [comment]
.html files wont work. The file you sent doesn't I cant download.

**Comment by bcrhbrhcdb** [comment]
you dont have to,

**Comment by enziviox** [comment]
OK delete the file from your computer and try opening it. Test it.

**Comment by Murat65536** [comment]
Sorry, but this extension won't work if you can't HTML files. You said something about trying to fix the `data:text/html,`. Is it broken on your end? From my understanding, it works for others and myself.

**Comment by Blobby-Boi** [comment]
if HTML files don't work, use the `data:` URL. If it refuses to open, replace `data:` with `data://`.

**Comment by Chattyice234** [comment]
For me data: URLS are blocked, blob: aren't

**Comment by Blobby-Boi** [comment]
Are HTML files blocked?

**Comment by Chattyice234** [comment]
For me, HTML files aren't blocked and don't have JS blocked. (yet but hopefully not)

**Comment by Blobby-Boi** [comment]
then you should be able to download `index.html` and run it

**Comment by Chattyice234** [comment]
I'm not the one having the problem with JS being blocked, the hanger works fine for me, it doesn't for the others on this issue

**Comment by Blobby-Boi** [comment]
ok

**Comment by minidogg** [comment]
> Can't use html files. Sorry to open another issue. Try to fix the data:text or make a github.io version please and thank you.

Are you still having this issue, or have you found a solution? My recommendation would be to go through sites you know aren't blocked and find one with some way to code using HTML or JavaScript. For example, National History Day website editor

**Comment by Chattyice234** [comment]
now my school blocked downloads but not ctrl + s, I can download sites that way as long as .js files arent separate from the html file

**Comment by minidogg** [comment]
> now my school blocked downloads but not ctrl + s, I can download sites that way as long as .js files arent separate from the html file

smart
